src/cmr/concepts/citation.js

```diff
@@ -0,0 +1,85 @@
+import Concept from "./concept";
+
+export default class Citation extends Concept {
+  constructor(headers, requestInfo, params) {
+    super('citations', headers, requestInfo, params)
+  }
+
+  /**
+   * Parse and return the array of data from the nested response body
+   * @param {Object} jsonResponse HTTP response from the CMR endpoint
+   */
+  parseJsonBody(jsonResponse) {
+    const { data } = jsonResponse
+
+    const { items } = data
+
+    return items
+  }
+
+  /**
+   * Returns an array of keys representing supported search params for the json endpoint
+   */
+  getPermittedJsonSearchParams() {
+    return [
+      ...super.getPermittedJsonSearchParams(),
+      'author-name',
```

Comment:
Wanted to confirm the hyphens are intentional.

Reply:
They were not, that should be `_`. Fixed.